homeassistant/components/denonavr/__init__.py

```diff
@@ -57,6 +64,19 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
 
+    async def disconnect(event: Event) -> None:
```

Comment:
```suggestion
    async def _async_disconnect(event: Event) -> None:
```

Since Home Assistant has both sync and async function in the codebase we prefix ones that are safe to run in the event loop and do not do any blocking I/O with `async_`

Reply:
Seems reasonable, I copied this from /homeassistant/components/devolo_home_network/__init__.py so I might submit a code cleanup PR later for a few other integrations that don't seem to follow this pattern.

Anyway, addressed in https://github.com/home-assistant/core/pull/85980/commits/f5e8b057d1716a236dcc1062734a5688d40a725e